portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,16 +39,8 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-    String quantityChosen = request.getParameter("number");
     String commentOrder = request.getParameter("order");
-    int amount;
     
-    try {
-      amount = Integer.parseInt(quantityChosen);
-    } catch (NumberFormatException e) {
-      amount = 1;
-    }
-
     Query query;
     if (commentOrder != null && commentOrder.equals("newest")) {
```

Comment:
This can be cleaned up with a variable and a ternary operator.

```java
boolean isDescending = ...
query = new Query(...).addSort("timestamp", isDescending ? SortDirection.Descending : SortDirection.Ascending);
```

Reply:
this is so much better, changed